src/SprykerSdk/AsyncApi/AsyncApi/Cli/AsyncApiCli.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\AsyncApi\AsyncApi\Cli;
+
+use Exception;
+use SprykerSdk\AsyncApi\AsyncApiConfig;
+use SprykerSdk\AsyncApi\Message\AsyncApiError;
+use SprykerSdk\AsyncApi\Message\AsyncApiInfo;
+use SprykerSdk\AsyncApi\Message\MessageBuilderInterface;
+use Symfony\Component\Process\Process;
+use Transfer\ValidateResponseTransfer;
+
+class AsyncApiCli implements AsyncApiCliInterface
+{
+    /**
+     * @var string
+     */
+    public const ASYNCAPI_CLI_COMMAND = 'asyncapi';
+
+    /**
+     * @var string
+     */
+    public const ASYNCAPI_CLI_VALIDATE = 'validate';
+
+    /**
+     * @var string
+     */
+    public const ASYNCAPI_CLI_VERSION = '--version';
+
+    /**
+     * @var \SprykerSdk\AsyncApi\AsyncApiConfig
+     */
+    protected AsyncApiConfig $config;
+
+    /**
+     * @var \SprykerSdk\AsyncApi\Message\MessageBuilderInterface
+     */
+    protected MessageBuilderInterface $messageBuilder;
+
+    /**
+     * @param \SprykerSdk\AsyncApi\AsyncApiConfig $config
+     * @param \SprykerSdk\AsyncApi\Message\MessageBuilderInterface $messageBuilder
+     */
+    public function __construct(AsyncApiConfig $config, MessageBuilderInterface $messageBuilder)
+    {
+        $this->config = $config;
+        $this->messageBuilder = $messageBuilder;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isCliInstalled(): bool
+    {
+        try {
```

Comment:
Extract running. the process into a separate method. This method might need to be marked as ignored for the coverage report.